docs/executor_plugins.md

```diff
@@ -69,22 +69,31 @@ kind: ExecutorPlugin
 metadata:
   name: hello
 spec:
-  description: This is the "hello world" plugin
-  container:
-    command:
-      - python
-      - -c
-    image: python:alpine3.6
-    name: hello-executor-plugin
-    ports:
-      - containerPort: 4355
+  sidecar:
+    container:
```

Comment:
`container` is defined under `sidecar`: 

https://github.com/argoproj/argo-workflows/blob/774bf47ee678ef31d27669f7d309dee1dd84340c/pkg/plugins/spec/plugin_types.go#L23-L29